frontend/src/properties/PropertiesWebSocketContainerMachine.ts

```diff
@@ -228,7 +228,7 @@ export const propertiesWebSocketContainerMachine = Machine<
       },
       isSelectionUnsupported: (_, event) => {
         const { selection, isRepresentation } = event as SwitchSelectionEvent;
-        return !selection || isRepresentation || selection.kind === 'Unknown' || selection.kind === 'Document';
+        return !selection || isRepresentation || selection.kind === 'Unknown' || selection.kind === 'Model';
```

Comment:
I would be in favor of keeping the old name to remove it entirely with the newly merged ADR not just change it at one or two very specific places.